docs/helm.md

```diff
@@ -0,0 +1,129 @@
+  recommend using a recent version of Helm if possible. Refer to the [Helm
+  documentation](https://helm.sh/docs/topics/version_skew/) for compatibility
+  between Helm and Kubernetes versions.
+* Add the Antrea Helm chart repository:
+
+  ```bash
+  helm repo add https://charts.antrea.io
+  helm repo update
+  ```
+
+## Charts
+
+### Antrea chart
+
+#### Installation
+
+To install the Antrea Helm chart, use the following command:
+
+```bash
+helm install antrea antrea/antrea --namespace kube-system
+```
+
+This will install the latest available version of Antrea. You can also install a
+specific version of Antrea (>= v1.8.0) with `--version <TAG>`.
+
+#### Upgrade
+
+To upgrade the Antrea Helm chart, use the following commands:
+
+```bash
+# Upgrading CRDs requires an extra step, see explanation below
+kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-crds.yml
+helm upgrade antrea antrea/antrea --version <TAG>
+```
+
+#### An important note on CRDs
+
+Helm 3 introduces "special treatment" for
+[CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/),
+with the ability to place CRD definitions (as plain YAML, not templated) in a
+special crds/ directory. When CRDs are defined this way, they will be installed
+before other resources (in case these other resources include CRs corresponding
+to these CRDs). CRDs defined this way will also never be deleted (to avoid
+accidental deletion of user-defined CRs) and will also never be upgraded (in
+case the chart author didn't ensure that the upgrade was
+backwards-compatible). The rationale for all of this is described in details in
+this [Helm community
+document](https://github.com/helm/community/blob/main/hips/hip-0011.md).
+
+Even though Antrea follows a [strict versioning policy](versioning.md), which
+reduces the likelihood of a serious issue when upgrading Antrea, we have decided
+to follow Helm best practices when it comes to CRDs. It means that an extra step
+is required for upgrading the chart:
+
+```bash
+kubectl apply -f https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea-crds.yml
+```
+
+When upgrading CRDs in production, it is recommended to make a back up of your
+Custom Resources (CRs) first.
+
+### Flow Aggregator chart
+
+The Flow Aggregator is on the same release schedule as Antrea. Please ensure
+that you use the same released version for the Flow Aggregator chart as for the
+Antrea chart.
+
+#### Installation
+
+To install the Flow Aggregator Helm chart, use the following command:
+
+```bash
+helm install flow-aggregator antrea/flow-aggregator --namespace flow-aggregator --create-namespace
+```
+
+This will install the latest available version of the Flow Aggregator. You can
+also install a specific version (>= v1.8.0) with `--version <TAG>`.
+
+#### Upgrade
+
+To upgrade the Antrea Helm chart, use the following command:
+
+```bash
+helm upgrade antrea antrea/antrea --version <TAG>
```

Comment:
Replace antrea with Flow Aggregator, and also add `--namespace flow-aggregator` to the command?